jina/peapods/zmq/__init__.py

```diff
@@ -370,8 +370,10 @@ def close(self):
             This method is idempotent.
         """
         if self.in_sock_type == zmq.DEALER:
-            print('### return cancel request', self.name)
-            self._send_cancel_to_router()
+            try:
+                self._send_cancel_to_router()
+            except:
+                pass  # in case the router is down already it throws an exception which can be silently ignored
```

Comment:
I would only silence TimeOutError which may be a normal way to end, but the other exceptions I would still log them. What have u been seing?